.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: kind/feature
+assignees: ''
+
+---
+
+<!-- Thank you for your contribution. Before you submit the issue:
+1. Search open and closed issues for duplicates.
+2. Read the contributing guidelines.
+-->
+
+**Description**
+
+<!-- Provide a clear and concise description of the feature. -->
+
+**Reasons**
+
+<!-- Explain why we should add this feature. Provide use cases to illustrate its benefits. -->
+
+**DoD:**
+- [ ] provide documentation
+- [ ] provide unit tests
+- [ ] test on SKR
+- [ ] verify [Get started guides](https://kyma-project.io/docs/kyma/latest/02-get-started/) 
+- [ ] verify resource limits
+- [ ] followup issue
+
```

Comment:
`bump` should be also added in the future, atm we don't know where

Reply:
so I am leaving it as it is